app/test_split_integer.py

```diff
@@ -2,20 +2,27 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert sum(split_integer(20, 6)) == 20
+    assert sum(split_integer(8, 2)) == 8,\
+        "Sum of the parts should be equal to start integer value"
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    assert split_integer(4, 2)[0] == split_integer(4, 2)[1]
+    assert split_integer(8, 2)[0] == split_integer(8, 2)[1],\
+        "Parts should be equal when value divisible by parts"
 
 
 def test_should_return_part_equals_to_value_when_split_into_one_part() -> None:
-    pass
+    assert split_integer(8, 1)[0] == 8, \
```

Comment:
it's generally good to verify the entire output of a function in your test
assert split_integer(8, 1) == [8]